java/src/Ice/src/main/java/com/zeroc/Ice/OutgoingAsync.java

```diff
@@ -35,7 +35,10 @@ public OutgoingAsync(
             }
         }
         if (_os == null) {
-            _os = new OutputStream(_instance, Protocol.currentProtocolEncoding);
+            _os =
+                    new OutputStream(
+                            Protocol.currentProtocolEncoding,
+                            _instance.defaultsAndOverrides().defaultFormat);
```

Comment:
My comment is incorrect - this code is a bit confusing and deserves a comment.

Even though the encoding is 1.0, we need to set the format in case the later encapsulation is 1.1. And this encapsulation will "inherit" this format set in the constructor.